Tests/IntegrationTests/start-system-under-test-docker.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+set -e
+
+ARCH=$(uname -m)
+if [ "$ARCH" = "arm64" ]; then
+  export DB_IMAGE=arm64v8/mysql:8
+elif [ "$ARCH" = "x86_64" ]; then
+  export DB_IMAGE=mysql:8
+else
+  echo "Unknown architecture"
+  exit 1
+fi
+
+function dc() {
+    if [ -n "$(docker compose version)" ]; then
+        # use the docker composer plugin
+        docker compose -f ./Tests/IntegrationTests/docker-compose.system-under-test.yaml $@
+    else
+        # legacy docker compose standalone
+        docker-compose -f ./Tests/IntegrationTests/docker-compose.system-under-test.yaml $@
+    fi
+}
+
+echo "#############################################################################"
+echo "# Start docker environment...                                               #"
+echo "#############################################################################"
+dc down
+dc up -d
+dc exec -T php bash <<-'BASH'
+    rm -rf /usr/src/app/*
+BASH
+docker cp "$(pwd)"/Tests/IntegrationTests/. "$(dc ps -q php)":/usr/src/app
+sleep 2
+
+echo ""
+echo "#############################################################################"
+echo "# Install dependencies...                                                   #"
+echo "#############################################################################"
+dc exec -T php bash <<-'BASH'
+    cd /usr/src/app
+    sudo chown -R docker:docker .
+    sudo chown -R docker:docker /home/circleci/
+    cd TestDistribution
+    composer install
+BASH
+
+echo ""
+echo "#############################################################################"
+echo "# Create sym links to mounted Docker volumes...                             #"
+echo "#############################################################################"
+echo ""
+dc exec -T php bash <<-'BASH'
+    # replace installed Neos Ui with local dev via sym link to mounted volume
+    # WHY: We want changes of dev to appear in system under test without rebuilding the whole system
+    rm -rf Packages/Application/Neos.Neos.Ui
+    ln -s /usr/src/neos-ui/ /usr/src/app/TestDistribution/Packages/Application/Neos.Neos.Ui
```

Comment:
This works in generally, but not in this flow. Later i think due to the `composer reinstall neos/test-site` this linking will be gone and we have the original composer neos ui... so this step should come later?

Reply:
I used this setup and it worked for me.

Thanks for fixing the issues on Linux ❤️

Reply:
Well this is not fixed yet ... and I don't know why it would work on your machine ... I mean the problem is within docker after all